src/katgpucbf/fgpu/main.py

```diff
@@ -422,7 +422,9 @@ def parse_args(arglist: Sequence[str] | None = None) -> argparse.Namespace:
         help="Mask off bottom bits of timestamp (workaround for broken digitiser)",
     )
     parser.add_argument(
-        "--use-vkgdr", action="store_true", help="Assemble chunks directly in GPU memory (requires Vulkan)"
+        "--use-vkgdr",
+        action="store_true",
+        help="Assemble chunks directly in GPU memory (requires sufficient GPU BAR space)",
```

Comment:
I think we can all agree the argument `--use-vkgdr` is pretty unambiguous as it is. _But_, still worth asking
* Can you adjust GPU BAR space without Vulkan?
* Or rather, you can adjust it without Vulkan, it just makes more/most sense to do it with a package that takes advantage of it (?)

Reply:
The amount of BAR space cannot be adjusted at all. Vulkan just lets you use the available space.